src/main/java/org/dcsa/ovs/notifications/service/impl/TimestampNotificationMailServiceImpl.java

```diff
@@ -328,4 +328,8 @@ public String apply(MatchResult matchResult) {
             super(key);
         }
     }
+
+    private Mono<TimestampDefinition> mapTimestampTypeName(OperationsEvent oe) {

```

Comment:
Sub-optimal name. Maybe something like `loadTimestampDefinition` (or maybe just inline it since it is a single line)

Reply:
I have in-lined the function, thanks for the feedback :)